BlazorCore/DSD.MSS.Blazor.Components.Core/Models/CommentDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DSD.MSS.Blazor.Components.Core.Models
+{
+    class CommentDTO
```

Comment:
I'd rename this as we don't need the DTO tag it's not used as a DTO. Also would add a descriptive comment.
```suggestion
    class Comment
```